tests/CMakeLists.txt

```diff
@@ -2,43 +2,58 @@ cmake_minimum_required(VERSION 2.8.7)
 
 project(GSLTests CXX)
 
-if (NOT EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/unittest-cpp/tests)
-    execute_process(COMMAND git submodule update --init WORKING_DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}")
+# will make visual studio generated project group files
+set_property(GLOBAL PROPERTY USE_FOLDERS ON)
+
+if (NOT EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/unittest-cpp/CMakeLists.txt)
+    find_package(Git)
+    execute_process(
+        COMMAND ${GIT_EXECUTABLE} submodule update --init
+        WORKING_DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}"
+    )
 endif()
 
 add_subdirectory(unittest-cpp)
 
 include_directories(
-    ../include
     ./unittest-cpp
```

Comment:
Why not `target_include_directories(gsl_tests_config INTERFACE ./unittest-cpp)`?